netci.groovy

```diff
@@ -152,7 +152,7 @@ set TMP=%TEMP%
     }
   }
 
-  def triggerPhraseOnly = true
+  def triggerPhraseOnly = false
```

Comment:
Causes the determinism suite to run at every PR. This bug is evidence it should be doing that.

i'd planned on turnig this on long ago, just forgot about it.